### PR TITLE
fix: Added changes to fix the lcov issue

### PR DESCRIPTION
**Pull request by ravali-rimmalapudi**
# Fixes #

Added changes to fix the lcov issue.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli-core/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: .github/workflows/cli-core-test.yml [hunk 1]
```diff
@@ -22,21 +25,14 @@ jobs:
...
-          fetch-depth: 0
       - name: SonarCloud Scan
         uses: sonarsource/sonarcloud-github-action@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
   notify-complete-fail:
-    if: ${{ failure() || cancelled() }}
-    needs: [ test, sonarcloud ]
+    if: ${{ failure() && github.ref == 'refs/heads/main' && github.event_name != 'pull_request'  }}
```
Comment (shrutiburman) [inline .github/workflows/cli-core-test.yml:34]: Shouldn't we also notify in case of a cancelled workflow?

Comment (ravali-rimmalapudi) [reply .github/workflows/cli-core-test.yml:34]: I felt If someone cancels a workflow run intentionally we don't need to notify in slack. Currently for travis also we are not doing this. Please let me know if you think `cancelled` is good to have I can add it back.

**Review by kridai — DISMISSED** [review]

File: .github/workflows/cli-core-test.yml (hunk 1 above)
Comment (shrutiburman) [reply .github/workflows/cli-core-test.yml:34]: That makes sense, if cancelled can only be triggered manually, there's no need to notify for it.

**Review by shrutiburman — APPROVED** [review]

**Review by kridai — DISMISSED** [review]

**Review by shrutiburman — DISMISSED** [review]

**Review by shamantraghav — APPROVED** [review]

**Comment by ravali-rimmalapudi** [comment]
image

**Review by shrutiburman — APPROVED** [review]

**Pull request merged**